cmd/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Println("Golang Bootcamp Shell")
+	fmt.Println("---------------------")
+
+	for {
+		fmt.Print("-> ")
+		input, _ := reader.ReadString('\n')
+		// convert CRLF to LF
+		input = strings.Replace(input, "\n", "", -1)
+
+		if input != "" {
+			result, e := handle(input)
+			if e == nil {
+				fmt.Println(result)
+			} else {
+				fmt.Println(e.Error())
+			}
+		}
+
+	}
+
+}
+
+func handle(input string) (result string, e error) {
+
+	r := regexp.MustCompile("\\s+")
```

Comment:
regexp will be not very efficient in complex or long input.
For simple requirement, better not over engineering.
in this case, we can use `strings.Fields`
https://golang.org/pkg/strings/#Fields

Reply:
thanks u :D